docker/config.deco.json

```diff
@@ -1,10 +1,12 @@
 { 
   "listenAddress": ":8080",
   "accounts": {
-    "spinup": {
+    "{{ .spot_account_id }}": {
       "region": "us-east-1",
       "akid": "{{ .spinup_akid }}",
       "secret": "{{ .spinup_secret }}",
+      "id": "{{ .spot_account_id }}",
+      "token": "{{ .spot_token }}"
```

Comment:
The `spinup` thing is just our internal name, you can leave it.  You should also remove the aws `akid` and `secret`, those were left there by mistake.

```json
    "spinup": {
      "region": "us-east-1",
      "id": "{{ .spot_account_id }}",
      "token": "{{ .spot_token }}"
    }
```

Reply:
cleaned up